### ISSUE TITLE
Does this support combos?

### ISSUE THREAD

**Issue by arcreigh** [Ataraksia/NeweggBot#18]
Newegg often has some great combo deals. Does this bot support those combo deals?
I like many others are trying to get our hands on ryzen 5000 series CPU's. In my case a brush with a failed pump has caused some heat danger to my current i7 4770k so I am finding that I am in desperate need of a replacement.

**Comment by Frosty8738** [comment]
If its an individual Newegg listing with its own product # then it should.

**Comment by arcreigh** [comment]
Yup i'd have to test but it doesn't look like they use the same URL schema either. I am trying to figure out the pull request as that will be a major fix. The bot in its current state without that PR is broken, however that PR is also missing a critical step in the ordering process. I have detailed my "failed" attempt and what I have gleaned from the code. I have done some work with node but puppeteer is completely new to me.

**Comment by Codolophier** [comment]
> however that PR is also missing a critical step in the ordering process

To employ the developers mantra, "It is working for me" :)  As outlined in my previous comment, the flow that I am seeing is as follows:

1. Shipping (default address specified, no selection needed)
2. Delivery (no selection needed)
3. Payment (no selection needed, just presentation of the CVV input)
4. Place Order button - click
5. Done - no further intervention needed

My apologies for not having availability today for a Discord chat to see what you are experiencing.  Can anyone else chime in on the flow they are seeing - with an in-stock item?

**Comment by Codolophier** [comment]
> Newegg often has some great combo deals. Does this bot support those combo deals?

@arcreigh Yes, so long as the combo deal is its own item, for example Combo.4206668.  If you're talking about the "People who ordered _**X**_ also ordered **_Y_**, Click here to add to cart", then no.  See [Issue #16](https://github.com/Ataraksia/NeweggBot/issues/16) for a similar question and confirmation.
